ligo-webide-frontend/base-components/packages/file-ops/src/filesystems/README.md

```diff
@@ -0,0 +1,15 @@
+The local filesystem code here is based on the local filesystem implemented by
+the Ethereum Remix project, which is licensed under the MIT license. The
+license text from the Remix project is reproduced verbatim here, in LICENSE.
+Note that the term "Contributors" in LICENSE refers to the Ethereum Remix
+contributors.
+
+Any changes to the code are still licensed under GPL v3, like the rest of this
+project. To understand which parts are part of the Ethereum Remix project, and
+which parts are novel to this project, please compare
```

Comment:
What if I also took different parts of code from Remix here and there? I should add them here?

Reply:
As we discussed in the meeting, you could change this MR and update it with the other places you changed, or add those changes to another MR. Which do you want to do?